core/src/test/scala/unit/kafka/server/ConsumerGroupHeartbeatRequestTest.scala

```diff
@@ -36,7 +37,7 @@ import scala.jdk.CollectionConverters._
 class ConsumerGroupHeartbeatRequestTest(cluster: ClusterInstance) {
 
   @ClusterTest()
-  def testConsumerGroupHeartbeatIsAccessibleWhenEnabled(): Unit = {
+  def testConsumerGroupHeartbeatIsInaccessibleWhenDisabledByStaticConfig(): Unit = {
```

Comment:
which config is the one that enables it? the `group.coordinator.rebalance.protocols` config?

Reply:
Correct.

Reply:
I see your answer below :)